packages/design-system/src/components/Combobox/Combobox.tsx

```diff
@@ -428,13 +435,25 @@ const ComboboxCreateItem = styled(ComboboxPrimitive.CreateItem)`
   &&[data-highlighted] {
     background-color: transparent;
   }
+  &&[data-disabled] {
+    color: ${({ theme }) => theme.colors.neutral600};
+    background: ${({ theme }) => theme.colors.neutral150};
+    cursor: not-allowed;
+  }
```

Comment:
there's a small visual issue with the border radii
![screenshot 2025-05-05 at 11 12 13@2x](https://github.com/user-attachments/assets/2f34d0ee-b718-4f2d-8a06-afd4f05008f6)